### ISSUE TITLE
Use PHPCSExtra once 1.0.0 version comes out

### ISSUE THREAD

**Issue by dingo-d** [infinum/eightshift-coding-standards#33]
[PHPCSExtra](https://github.com/PHPCSStandards/PHPCSExtra) is a great additional standard that uses some improved sniffs.

One of such is the [NormalizedArrays.Arrays.ArrayBraceSpacing](https://github.com/PHPCSStandards/PHPCSExtra#normalizedarraysarraysarraybracespacing-wrench-bar_chart-books) sniff, which should help us with the array spacing inside the brackets issue:

```php
[ 'foo' ] // Should be caught by PHPCS as an error -> ['foo'].
```
